python/ray/rllib/policy_gradient/policy_gradient.py

```diff
@@ -224,6 +224,14 @@ def train(self):
         print("total time so far:", time.time() - self.start_time)
 
         result = TrainingResult(
-            self.experiment_id.hex, j, total_reward, traj_len_mean, info)
+            self.experiment_id.hex, j, total_reward, traj_len_mean,
+            checkpoint_path, info)
 
         return result
+
+    def restore(self, checkpoint_path):
+        self.saver.restore(self.model.sess, checkpoint_path)
+
+    def compute_action(self, observation):
```

Comment:
Yes, (and DQN) it's nontrivial though. I can leave a TODO here to note it as a task.